### PR TITLE
Naxxus 2

### PR DESCRIPTION
**Pull request by DarkWorldsArtist**
### Other checks:

-   [ ] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/lib/data/creatables/bsoItems.ts [hunk 1]
```diff
@@ -878,6 +878,52 @@ const bsoItems: Createable[] = [
 		outputItems: {
 			[itemID('Crystal fishing rod')]: 1
 		}
+	},
+	{
+		name: 'Void Staff (u)',
```
Comment (gc) [inline src/lib/data/creatables/bsoItems.ts:883]: Items (in all of this PR) shouldn't be cased like this, in rs only the first word has a capital letter. im not that bothered though, you dont have to change them.

File: src/lib/minions/data/killableMonsters/custom/bosses/Naxxus.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import { Time } from 'e';
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
+import { naxxusCL } from '../../../../../data/CollectionsExport';
+import { GearStat } from '../../../../../gear';
+import { KillableMonster } from '../../../../../minions/types';
+import { makeKillTable } from '../../../../../util/setCustomMonster';
+
+const runes = new LootTable()
```
Comment (gc) [inline src/lib/minions/data/killableMonsters/custom/bosses/Naxxus.ts:9]: some measurements/simulations on the loot would be good, like how much per hour, maybe how much loot you'd get finishing the cl

File: src/lib/util/brewRestoreSupplyCalc.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+				hasEnough: false,
+				foodBank: itemBank,
+				foodReason: `${user.username} does not have enough saradomin brews. ${enhancedBrews[1]} enhanced & ${brews[1]} normal found, ${brewsNeeded} required (enhanced count for 2).`
+			};
+		}
+	}
+
+	if (!restoresNeeded) restoresNeeded = Math.max(1, Math.floor(brewsNeeded / 3));
+	let totalRestores = 0;
+	const enhancedRestores = userItems.filter(([item]) => item.name.toLowerCase() === 'enhanced super restore')[0] ?? [
```
Comment (gc) [inline src/lib/util/brewRestoreSupplyCalc.ts:37]: dont do this.  use `itemID(name)`

File: src/lib/util/brewRestoreSupplyCalc.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+		'',
+		0
+	];
+	const restores = userItems.filter(([item]) => item.name.toLowerCase() === 'super restore(4)')[0] ?? ['', 0];
```
Comment (gc) [inline src/lib/util/brewRestoreSupplyCalc.ts:41]: same here

**Review by gc — CHANGES_REQUESTED** [review]

**Comment by themrrobert** [comment]
this is the totally broken PR, i'm closing it, please open a new one with the right branch.

**Pull request closed without merging**